docs/sources/lids/_DRAFT_RemoteRuleEvaluation.md

```diff
@@ -0,0 +1,84 @@
+
+**Pros:**
+- Takes full advantage of Loki's query acceleration techniques, leading to faster and more efficient rule evaluation
+- Operationally simple as existing `query-frontend`/`query-scheduler`/`querier` setup can be used
+- Per-tenant isolation available in Loki's query path (shuffle-sharding, per-tenant queues) can be used to reduce or eliminate the noisy neighbour problem
+
+**Cons:**
+- Increased interdependence in components, increased cross-component networking
+- Reusing the same `query-frontend`/`query-scheduler`/`querier` setup can cause expensive queries to starve rule evaluations of query resources, and vice versa
+  - Additional complexity introduced if this setup needs to be duplicated for rule evaluations (recommended)
+
+## Other Notes
+
+If this feature were to be used in conjunction with [rule-based sharding](https://github.com/grafana/loki/pull/8092), this can present some further optimisation but also some additional challenges to consider.
+
+> Aside: the `ruler` shards by rule group by default, which means that rules can be unevenly balanced across `ruler` instances if some rule groups have more expensive queries than others. Another consequence of this is that rule groups execute sequentially, so expensive queries can cause subsequent rules in the group to be delayed or even missed. Rule groups are evaluated concurrently.
+
+Rule-based sharding distributes rules evenly across all available `ruler` instances, each in their own rule group. Consequentially, each rule that belongs to a `ruler` instance will be evaluated concurrently (as they're each in their own rule group). For tenants with hundreds or thousands of rules, this can result in large batches of queries being sent to the `query-frontend` in quick succession, should they all use the same interval or happen to overlap.
+
+Care must be taken by operators who run large multi-tenant setups to ensure that large volumes of queries can be received, queued, and processed in an acceptable timeframe. The `query-scheduler` component is highly recommended in these situations, as it will enable the `query-frontend` and `query` components to scale out to accommodate the load. Shuffle-sharding should also be implemented to ensure that tenants with particularly large workloads do not starve out the query resources of other tenants. Alerting should also be put in place to notify operators if rule evaluations are being routinely missed or a tenant's query queue becomes full.
+
+Rule-based sharding and remote evaluation are _mutually exclusive_. It is recommended that operators first implement remote evaluation to improve `ruler` reliability, and _then_ further investigate rule-based sharding if rule evaluations are still being missed due to the sequential execution of rule groups, or advise their tenants to split these rule groups up.
```

Comment:
I don't understand the first sentence here. Why are they mutually exclusive, when you later describe that you first should enable remove evaluation, and then rule-based sharding.

Did you mean that you shall not enable both simultaneously without prior testing of one or another?

Reply:
Yeah, I should clarify this more. I meant that they are separate features and one does not require the other. Good call-out, I'll tighten up the language here.